chaos-bugbounty-list.json

```diff
@@ -1728,6 +1728,14 @@
          "domains":[
             "wickr.com"
          ]
+      },
+      {
+         "name": "Yatra",
+         "url": "https://www.yatra.com/online/bug-bounty",
+         "bounty": true,
+         "domains":[
+            "www.yatra.com"
```

Comment:
Can you please replace the `www.yatra.com` with `yatra.com` as the domain name is expected input.

Reply:
Done as per request!